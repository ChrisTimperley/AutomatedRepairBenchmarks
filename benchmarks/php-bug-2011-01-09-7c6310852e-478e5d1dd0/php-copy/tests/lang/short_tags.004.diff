012+ <?= "{$b}"?>
012- 
013- Notice: Undefined variable: b in %s on line %d