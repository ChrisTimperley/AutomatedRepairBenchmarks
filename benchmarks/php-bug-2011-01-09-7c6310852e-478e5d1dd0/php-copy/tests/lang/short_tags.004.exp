<%= 'so should this' %>


<?= $a?>

<%= $a%>

<? $b=3; ?>


Notice: Undefined variable: b in %s on line %d

Notice: Undefined variable: b in %s on line %d