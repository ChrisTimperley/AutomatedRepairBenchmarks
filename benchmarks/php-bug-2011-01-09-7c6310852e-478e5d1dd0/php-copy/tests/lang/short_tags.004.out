<%= 'so should this' %>


<?= $a?>

<%= $a%>

<? $b=3; ?>


Notice: Undefined variable: b in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/lang/short_tags.004.php on line 14
<?= "{$b}"?>